backend/utils/Setting.ts

```diff
@@ -0,0 +1,786 @@
+'use strict';
```

Comment:
why we have this header ?

Reply:
Sorry was a leftover from the Etherpad repository.

Reply:
I suspected as much. That's what the review is for, to avoid keeping legacy code.

Reply:
Yeah I'm sometimes a bit blindfolded. So good catch :)